pkg/appmanager/routing.go

```diff
@@ -19,17 +19,16 @@ package appmanager
 import (
 	"bytes"
 	"fmt"
+	log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger"
 	"net/url"
 	"reflect"
 	"sort"
 	"strconv"
 	"strings"
-	"sync"
-
-	log "github.com/F5Networks/k8s-bigip-ctlr/pkg/vlogger"
 
 	routeapi "github.com/openshift/origin/pkg/route/api"
 	"k8s.io/client-go/pkg/apis/extensions/v1beta1"
+	"sync"
```

Comment:
Leave the import statements as they were. Standard go convention for import structure is:

1. standard library
2. third party
3. internal packages